roles/disaster_recovery/tasks/generate_mapping.yml

```diff
@@ -0,0 +1,6 @@
+- block:
+    - name: Generate mapping var file
+      command: python3 {{ role_path }}/files/generate_mapping.py -a "{{ site }}" -u "{{ username }}" -p "{{ password }}" -c "{{ ca }}" -f "{{ var_file }}"
```

Comment:
Watch out if the `role_path` is correct

Reply:
I will still need to test the full flow, so it will simply not work if the `role_path` is not correct. The more interesting question is whether it's correct to execute `python3` directly.

Reply:
yeah, I was just pointing out that it can fail here because it does not know it's under `roles/disaster_recovery`
(maybe I'm just missing something)

Reply:
I'm starting testing soon. But the question is should I test it on Python2 version? Because the current implementation calls the `python3` command...

Reply:
It depends where the role is going to be executed. If you want only EL8 support, then test Python 3 only. If you expect, that role will be executed even on older 4.3/4.2 hosts, then you need to support both Python 2 and Python 3

Reply:
I think there should be another PR for this and not do everything when migrating.